curriculum/challenges/english/25-front-end-development/lab-house-painting/66d6a7a3e1aa411e94bf2346.md

```diff
@@ -0,0 +1,469 @@
+---
+id: 66d6a7a3e1aa411e94bf2346
+title: Build a House Painting
+challengeType: 14
+dashedName: build-a-house-painting
+demoType: onClick
+---
+
+# --description--
+
+**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab. 
+
+**User Stories:**
+
+1. You should have a `#house` `div`.
+1. Your `#house` should have a `position` set to `relative` so its children can be positioned with respect to it.
+1. Your `#house` should have a width of `500px` and a height of `400px`.
+1. Your `#house` should have a background color and a border set.
+1. You should have a `#chimney`, `#decor`, `#window-1`, `#window-2`, and `#door` `div`s within the `#house`.
+1. All of the immediate children of the `#house` should have a `position` of `absolute`.
+1. All of the immediate children of the `#house` should have a width, height, border, and background color set.
```

Comment:
When I followed these instructions, I came up with this code, but it didn't pass:

```css
#house > * {
position: absolute;
background-color: pink;
width: 90px;
height: 100px;
border: 2px solid;
}
```
Either it should be mentioned in the instruction to target these `divs` separately or make the tests flexible. These properties are mutual in all the children of `#house`.

Reply:
The hints saying "you should target `#X` and set its `Y` property" should work for this.